2022/1xxx/CVE-2022-1091.json

```diff
@@ -1,18 +1,80 @@
 {
```

Comment:
Invalid source.

Reply:
I don't see what the issue is on that one oO